gsy_e_sdk/setups/Grid_Operator_API_scripts/rest_grid_fees.py

```diff
@@ -0,0 +1,113 @@
+# flake8: noqa
+# pylint: disable=duplicate-code
+
+"""
+Template file for markets management through the gsy-e-sdk api client using Redis.
+"""
+import os
+from time import sleep
+from gsy_e_sdk.aggregator import Aggregator
+from gsy_e_sdk.rest_market import RestMarketClient
+from gsy_e_sdk.utils import log_grid_fees_information
+from gsy_e_sdk.utils import get_area_uuid_from_area_name_and_collaboration_id
+
+current_dir = os.path.dirname(__file__)
+
+MARKET_NAMES = ["Grid", "Community"]
+ORACLE_NAME = "dso"
+SLOT_LENGTH = 15  # leave as is
+AUTOMATIC = True
+
+
+class Oracle(Aggregator):
+    """Class to represent the Grid Operator client type."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.is_finished = False
+
+    def on_market_cycle(self, market_info):
+        current_market_fee = {}
+        for area_uuid, area_dict in self.latest_grid_tree_flat.items():
+            if area_dict["area_name"] in MARKET_NAMES:
+                self.add_to_batch_commands.last_market_dso_stats(area_uuid)
+                current_market_fee[area_dict["area_name"]] = area_dict[
+                    "current_market_fee"
+                ]
+        next_market_fee = self.set_new_market_fee()
+        log_grid_fees_information(MARKET_NAMES, current_market_fee, next_market_fee)
+
+    def set_new_market_fee(self):
```

Comment:
NIT, as this method is only used inside the class, it can be private:
```suggestion
    def _set_new_market_fee(self):
```

Reply:
you decided to not implement this, right? Ok!